app/controllers/scenarios_controller.rb

```diff
@@ -95,6 +95,20 @@ def update
     end
   end
 
+  def enable_or_disable_all_agents
+    @scenario = current_user.scenarios.find(params[:id])
+
+    respond_to do |format|
+      if @scenario.agents.update_all disabled: params[:scenario][:disabled]
```

Comment:
@dsander for some reason `update_all` isn't working........but I think it should. Would you know anything that scenario model is hooked up with that potentially make it fail?